charmhelpers/contrib/openstack/vaultlocker.py

```diff
@@ -48,11 +52,30 @@ def __call__(self):
                     unit=unit,
                     rid=relation_id
                 )
+                token = hookenv.relation_get(
```

Comment:
There are three separate relation_gets here, why not retrieve all the relation data into a dict and then access that?